source/Common.Tests/Common.Tests.csproj

```diff
@@ -32,6 +34,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="LiteNetLib">
+      <HintPath>..\Coop.Tests\bin\Debug\LiteNetLib.dll</HintPath>
+    </Reference>
```

Comment:
Why not use nuget here?

Reply:
Removed common tests as there were no tests

Reply:
Good catch fixed